instance/models/utils.py

```diff
@@ -482,6 +484,49 @@ def purge(self):
         """
         return self._client.kv.delete(self.prefix, recurse=True)
 
+    def leave(self, force=True):
+        """
+        This method will request the agent to leave the cluster, if the
```

Comment:
I'm confused as to how these new functions will actually request a _particular_ agent to leave the cluster. There seems to be no option in these functions so far to allow that.